### PR TITLE
Adafruit_IO_Power_Relay: update to settings.toml

### PR DESCRIPTION
**Pull request by dhalbert**
Guide updated.

### CODE REVIEW

File: Adafruit_IO_Power_Relay/code.py [hunk 1]
```diff
@@ -107,8 +106,8 @@ def on_relay_msg(client, topic, value):
 # Set up a MiniMQTT Client
 client = MQTT.MQTT(
     broker="io.adafruit.com",
-    username=secrets["aio_username"],
-    password=secrets["aio_key"],
+    username=os.getenv("AIO_USERNAME"),
```
Comment (brentru) [inline Adafruit_IO_Power_Relay/code.py:109]: @dhalbert `aio_username` is changed to to `AIO_USERNAME` (uppercase here) as well as `aio_key`/`AIO_KEY`. 

Is this lcase to ucase intentional for the TOML files? Is it referenced by the uppercase variable name in the guide?

Comment (dhalbert) [reply Adafruit_IO_Power_Relay/code.py:109]: Uppercase is what we have been using as a standard. I changed the guides to use uppercase names.

There has been some discussion of `ADAFRUIT_AIO_*`, but `AIO_*` seems fine and is used in most guides.

Comment (brentru) [reply Adafruit_IO_Power_Relay/code.py:109]: > I changed the guides to use uppercase names.

Sounds good1

**Review by brentru — APPROVED** [review]
@dhalbert  Approved pending the two CI checks are resolved. You may merge in when ready.

**Comment by dhalbert** [comment]
Thanks! Build failures are due to an unrelated error elsewhere.

**Pull request merged**